### PR TITLE
Added filter by project in the statistics of the detailed reports.

### PR DESCRIPTION
**Pull request by jerdzica**
(Similarily to estimation reports).

### CODE REVIEW

File: addons/bestja_detailed_reports/views/detailed_reports_summary.xml [hunk 1]
```diff
@@ -65,6 +67,21 @@
                 ('detailed_report.state', '!=', 'draft'),
             ]</field>
             <field name="context">{
+                'search_default_responsible_project': active_id,
+                'search_default_accepted': True,
+            }</field>
+        </record>
+
+        <record model="ir.actions.act_window" id="action_detailed_reports_product_summary_lvl0">
```
Comment (KrzysiekJ) [inline addons/bestja_detailed_reports/views/detailed_reports_summary.xml]: This record is exactly the same as the one with the id `action_detailed_reports_product_summary_lvl1`. The `_compute_responsible_project` method returns from the same branch of code for values `0` and `1`. Menu entries are analogous for values `0` and `1`. What's the purpose of splitting this into two?

Comment (jerdzica) [inline addons/bestja_detailed_reports/views/detailed_reports_summary.xml]: They're not the same - one uses responsible_project and the other one top_project. The middle level organization needs to see its tonnage in the statistics, as well as the lower level organizations tonnage (hence uses responsible_project field). On the other hand, the top level organization wants to see only the sums for middle level organizations (for middle level organization sum of tonnage of this organization and its children; hence top_project).

Comment (jerdzica) [inline addons/bestja_detailed_reports/views/detailed_reports_summary.xml]: This pull request is a dual pull request to previous pull request on estimation reports.

Comment (KrzysiekJ) [inline addons/bestja_detailed_reports/views/detailed_reports_summary.xml]: You’re obviously right, it’s my mistake that I didn’t notice the search defaults difference.

**Comment by KrzysiekJ** [comment]
Merged in c23d3f5.

**Pull request closed without merging**